### ISSUE TITLE
Magento ver. 2.4.2 is not allowing me to view orders 

### ISSUE THREAD

**Issue by jadebaileywowcher** [magento/magento2#34773]
### Preconditions (*)

1.
2.

### Steps to reproduce (*)
Magento ver. 2.4.2 is not allowing me to view orders due to an error that flashed up from a previous search - it is not allowing me to do a new orders search through the option 'sales', 'orders' 
1.on the left i click sales, then click on orders and then i get an error that states "Attention Something went wrong." then the circle in the middle spins and nothing else happens

2. screenshot 
![image](https://user-images.githubusercontent.com/87760355/145195737-ab1b2d3b-719d-46ca-84a4-1d22a18a020f.png)

### Expected result (*)

1. [Screenshots, logs or description]
2. I expect to be able to search for order numbers and emails but the error is preventing me clicking in the search section and is greyed out by the spinning circle 

### Actual result (*)

1. [Screenshots, logs or description]
2.

---
Please provide [Severity](https://devdocs.magento.com/guides/v2.4/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [x] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by engcom-November** [comment]
Verified the issue on Magento 2.4-develop branch but cannot able to reproduce the issue.
Sales - Orders - Searched using Keywords - ID and email id - No issue. Filter applied as expected.
User is able to view Order details page - No issue
![image](https://user-images.githubusercontent.com/60198231/145199803-3f5b9687-98e7-4bde-b262-f69414ef87cb.png)

@jadebaileywowcher FYI.. Sometimes, this error can occur due to version upgrade. Check "catalog_product_entity" table in DB and verify if any Products have NULL as SKU value
SELECT * FROM catalog_product_entity WHERE sku IS NULL to check all product not have sku NULL, then update them.
Another temporary fix is to clear "ui_bookmark table".   > TRUNCATE ui_bookmark;
Make sure to clear cache and add permission/rights.
Kindly recheck the issue on Magento 2.4-develop branch and provide missing steps to reproduce if you are still facing the issue.
Thank you.

**Comment by jadebaileywowcher** [comment]
Im really sorry but im in customer services and have no technical knowledge of what to do to resolve the issue!? Do I need to speak to the tech guy in the office?

Thank you

Jade Bailey
EU Customer services

**Comment by engcom-November** [comment]
Hi @jadebaileywowcher,
Kindly apply the possible solutions that are provided in [comment](https://github.com/magento/magento2/issues/34773#issuecomment-988729837) in your local system. If its still not resolved, provide configuration details (Ex: Magento specific versions before and after upgrade if issue occurred due to upgrade).

Thank you.

**Comment by jadebaileywowcher** [comment]
Please can you put this into Laymans terms as I literally work in customer services - I do not have any knowledge of the set-up of the systems,  I just use them to do my job role. Please can you assist from the back end? Or dumb it down for me a bit.


Thanks

Jade

Wowcher customer services

**Comment by engcom-November** [comment]
Hi @jadebaileywowcher ,
Please check the below possible solutions to resolve your issue in local system:
1. Switch to your Magento Database - Verify Database table "catalog_product_entity" if any Products have NULL as SKU value
SQL Query for the same: > SELECT * FROM catalog_product_entity WHERE sku IS NULL 
Update the sku using UPDATE Query  > UPDATE catalog_product_entity SET sku='" WHERE sku IS NULL;

2. Another temporary fix is to clear "ui_bookmark table". 
SQL Query for the same > TRUNCATE ui_bookmark;

3. Make sure to clear cache and add necessary permission/rights from Magento root folder termnial.
> php bin/magento indexer:reindex
> php bin/magento cache:flush
> chmod -R 777 var/* generated/ pub/*
> chmod -R 777 /var/www/html/yourMagentoLocal_Folder

**Comment by engcom-November** [comment]
Hi @jadebaileywowcher , 
We are closing this issue as there has been no latest update on the same.
Thank you.

**Comment by jadebaileywowcher** [comment]
Morning,

This issue still persists, I am just unable to do the things you ask me to as I am working from home and not technical in the slightest.

Here are screenshots again

Magento does not allow me to view the orders section in the sales tab shown in this screenshot -
***@***.***

This next screenshot shows you what happens when I click on orders -

***@***.***

It advises something went WRONG with the processing - this has happened before and you guys sorted it out after a few weeks but this is still happening for my log in which is WowcherSupportTeam.

Please attempt to log in as this and you will see the same problem. I am able to use Magento still, but I could really do with the orders tab to be reset and working again.

Thanks

Jade

Wowcher EU Customer Service

**Comment by snadiminty** [comment]
Hi Jade!! Were you able to resolve this issue? I am having the same issue with Magento, have been trying to resolve this for over a month, to no avail!!

Can we get a step by step guide instead of the technical jargon, which is tough to understand? for example, how to we switch to our magento database?

**Comment by jadebaileywowcher** [comment]
Hi there, I just got a new log in to be honest so no I didn't get a resolve on the actual issue but the steps to follow were just not able to be followed by someone not technically trained.

:)
